x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -48,3 +56,44 @@ func (k Keeper) ValidateInbound(
 
 	return &cctx, nil
 }
+
+// CheckIfMigrationDeposit checks if the sender is a TSS address and returns an error if it is.
+// If the sender is an older TSS address, this means that it is a migration transfer, and we do not need to treat this as a deposit and process the CCTX
+func (k Keeper) CheckIfMigrationDeposit(ctx sdk.Context, msg *types.MsgVoteInbound) error {
+	additionalChains := k.GetAuthorityKeeper().GetAdditionalChainList(ctx)
+
+	historicalTssList := k.zetaObserverKeeper.GetAllTSS(ctx)
+	chain, found := k.zetaObserverKeeper.GetSupportedChainFromChainID(ctx, msg.SenderChainId)
+	if !found {
+		return observertypes.ErrSupportedChains.Wrapf("chain not found for chainID %d", msg.SenderChainId)
+	}
+
```

Comment:
the sender chain was not checked before in validating inbound, but I think it's safe to add it.

mainnet : https://zetachain.blockpi.network/lcd/v1/public/zeta-chain/observer/supportedChains
testnet : https://zetachain-testnet-api.itrocket.net/zeta-chain/observer/supportedChains